7-layouts/css/styles.css

```diff
@@ -0,0 +1,160 @@
+.img-container {
+    flex-wrap: wrap;
+}
+
+header {
+    background-color: #323233;
+    padding: 10px;
+    display: flex;
+    align-items: center;
+    color: white;
+    font-size: 25px;
+    justify-content: start;
+}
+
+.logo-fire {
+    width: 25px;
+    margin-right: 10px;
+}
+
+.banner-container {
+    width: 100%;
+    height: 35%;
+    position: relative;
+    box-sizing: border-box;
+    background-image: url('https://c.wallhere.com/photos/bf/60/cat_blue_eyes_face_animals_eyes-216035.jpg!d');
+    background-repeat: no-repeat;
+    background-size: cover;
+    background-position: center;
```

Comment:
group common properties in a single class so you don't have to repeat these properties for each image. You can do: `
.banner-bg-image {  
box-sizing: border-box;
    background-repeat: no-repeat;
    background-size: cover;
    background-position: center;
}`